### PR TITLE
Completed html-tables exercise.

### PR DESCRIPTION
**Pull request by santiagolism**
Screenshot (365)

### CODE REVIEW

File: html-tables/html-tables/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      </tr>
+     </thead>
+     <tbody>
+      <tr>
+    <td> Grace Hopper</td>
+      <td>Computer Science</td>
+      <td>100</td>
+  </tr>
+  <tr>
+      <td>Alan Turing</td>
```
Comment (BrettAlbright) [inline html-tables/html-tables/index.html]: notice how the indentation is shifting here. you want to make sure to keep the indentation even between sibling elements as it makes the code much easier to read.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Small formatting update!

**Comment by santiagolism** [comment]
Screenshot (366)

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**